self-core-impl/src/main/java/com/selfxdsd/core/GithubPersonalRepos.java

```diff
@@ -83,7 +83,7 @@
     }
 
     /**
-     * @todo #1166:60min Possible bug in class FromHeaders. The bellow method
+     * TODO #1166:60min Possible bug in class FromHeaders. The bellow method
```

Comment:
@amihaiemil here checkstyle complained about not starting the sentence with a letter.